include/extractor/scripting_environment_lua.hpp

```diff
@@ -49,8 +52,8 @@ struct LuaScriptingContext final
 class Sol2ScriptingEnvironment final : public ScriptingEnvironment
 {
   public:
-    static const constexpr int SUPPORTED_MIN_API_VERSION = 0;
-    static const constexpr int SUPPORTED_MAX_API_VERSION = 1;
+    static const constexpr int SUPPORTED_MIN_API_VERSION = 1;
```

Comment:
Does that mean profiles with version `0` won't work anymore? That would be consider a breaking change, and I would like to avoid this.